_posts/2022-11-25-open-mic-pipeline.markdown

```diff
@@ -0,0 +1,19 @@
+---
+title: "Developing RDF-based data processing pipelines using SPipes"
+categories: [Open Mic Session, Open Mic]
+excerpt: "SPipes is engine to execute RDF-based data processing pipelines. The goal of the session is to present current state of the SPipes and show practical tips  to develop and deploy SPipes scripts."
+---
+
+{% include figure image_path="assets/images/openmics/2022-11-25-spipes.png" alt="Miro Blaško - Developing RDF-based data processing pipelines using SPipes" %}{: .post-image }
+
+On the 25th November, speaker [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško) took presentation called Developing RDF-based data processing pipelines using SPipes.
+
+{% include video id="JxN26VEzmtw" provider="youtube" %}
+
+SPipes (https://github.com/kbss-cvut/s-pipes) (Semantic data pipelines) is RDF-based scripting language based on SPARQL motion. It defines data pipelines in the form of an acyclic oriented graph of modules expressed in RDF. Concrete modules are implemented typically in Java while executions of modules are parametrized within RDF. The goal of the session is to explain strong and weak parts of the language. Moreover, it also containes practical tips on how to develop and deploy SPipes pipelines using SPipes engine that was developed at KBSS.
+
+{% include figure image_path="/assets/images/posts/spipes-model.png" alt="SPipes model" %}{: .post-image }
```

Comment:
We do not support svg? I have sent you SVG and you created PNG. BTW, there is selection of text in the image (word "next" is selected).